### ISSUE TITLE
Accessibility Report: Logo Component

### ISSUE THREAD

**Issue by sherakama** [SU-SWS/decanter#322]
Use the AMP tool to scan the [Logo](http://decanter.stanford.edu/section-components.html#kssref-components-logo) component for accessibility issues and document them to this ticket.

**Comment by sherakama** [comment]
AMP Report: https://stanford.levelaccess.net/public/reporting/view_report.php?report_id=283424

No violations found.

**Comment by sherakama** [comment]
Chrome Audit Scan: 100%

![Screen Shot 2019-03-13 at 8 56 37 PM](https://user-images.githubusercontent.com/550602/54330333-88ad0580-45d2-11e9-8c24-b7c850623819.png)
